.github/ISSUE_TEMPLATE/suggested-feature.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Suggested Feature
```

Comment:
Instead of this add a heading for bug / feature / question and let them add the corresponding answer under it.